vividus-bdd-engine/src/main/java/org/vividus/bdd/expression/RoundExpressionProcessor.java

```diff
@@ -24,13 +24,16 @@
 
 import javax.inject.Named;
 
+import org.apache.commons.lang3.EnumUtils;
 import org.apache.commons.lang3.math.NumberUtils;
 
 @Named
 public class RoundExpressionProcessor implements IExpressionProcessor
 {
     private static final Pattern ROUND_EXPRESSION_PATTERN = Pattern.compile(
-            "^round(?:\\((-?\\d+(?:\\.\\d*)?)(?:,\\s*(\\d+))?\\))$", Pattern.CASE_INSENSITIVE);
+            "^round(?:\\((-?\\d+(?:\\.\\d*)?)(?:,\\s*(\\d+))?(?:,\\s*"
+                    + "(ceiling|floor|up|down|half_up|half_down|half_even|unnecessary))?\\))$",
```

Comment:
build this string based on `java.math.RoundingMode#values()` so that if new values appears in the next Java versions, it won't be required to update the string